pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -176,6 +176,21 @@ func (pl *VolumeRestrictions) PreFilter(ctx context.Context, cycleState *framewo
 	if err != nil {
 		return nil, framework.AsStatus(err)
 	}
+
+	needCheck := false
+	for i := range pod.Spec.Volumes {
+		v := &pod.Spec.Volumes[i]
+		// fast path if there is no conflict checking targets.
+		if v.GCEPersistentDisk == nil && v.AWSElasticBlockStore == nil && v.RBD == nil && v.ISCSI == nil {
```

Comment:
make this an util function here? It may looks like `func noneConflictVolume(v1.Volume) bool`

Reply:
updated.